teku/src/test/java/tech/pegasys/teku/cli/subcommand/ValidatorClientCommandTest.java

```diff
@@ -210,6 +210,23 @@ public void clientExecutorThreadsShouldThrowOverLimit() {
             "--Xvalidator-client-executor-threads must be greater than 0 and less than 5000.");
   }
 
+  @Test
+  public void shouldSetUseDvtSelectionsEndpoint() {
+    final String[] args = {"vc", "--network", "minimal", "--Xdvt-integration-enabled"};
+    final TekuConfiguration config = getTekuConfigurationFromArguments(args);
+
+    assertThat(config.validatorClient().getValidatorConfig().isDvtSelectionsEndpointEnabled())
+        .isTrue();
+  }
+
+  @Test
+  public void shouldNotUseDvtSelectionsEndpointByDefault() {
+    final String[] args = {"vc", "--network", "minimal"};
```

Comment:
again, shouldnt need to specify network? if it IS needed, i'd expect us to have a mainnet default on the vc like we do on the main command.